src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -367,12 +367,25 @@ impl MDARegions {
     }
 }
 
+#[derive(Debug, Default, PartialEq, PartialOrd)]
```

Comment:
Please add a doc comment, like:

A type representing the actual size of variable length metadata written within this
metadata region. This amount must never be greater than the size of the region allocated for the variable length metadata, which has type MDADataSize.